lib/install.ps1

```diff
@@ -632,7 +632,7 @@ function cookie_header($cookies) {
 function is_in_dir($dir, $check) {
     $check = "$(fullpath $check)"
     $dir = "$(fullpath $dir)"
-    $check -match "^$([regex]::escape("$dir"))(\\|`$)"
+    $check -match "^$([regex]::Escape("$dir"))([/\\]|$)"
```

Comment:
`` ` `` is missing, no?

Reply:
I found the regex works fine without `` ` `` and removed it.

Reply:
I think we should leave it in for more clarity

Reply:
OK